app/api/events.py

```diff
@@ -206,6 +236,76 @@ def get_event_participants(request: Request, id: str, token: AccessTokenPayload
     return [{'id': p['id'], 'name': p['realName']} for p in event['participants']]
 
 
+@router.get('/{id}/options', dependencies=[Depends(validate_uuid)])
+def get_event_options(request: Request, id: str, token: AccessTokenPayload = Depends(authorize)):
+    """ Get event options for requesting user """
+    db = get_database(request)
+    event = get_event_or_404(db, id)
+
+    # Get event (if user is joined)
+    user_event = db.events.find_one({"eid": event["eid"]}, {"participants": {
+        "$elemMatch": {"id": UUID(token.user_id)}}})
+    
+    # Check whether user exists in event
+    if not user_event:
+        raise HTTPException(400, "User not joined event!")
+
+    try:
```

Comment:
I think this block can be merged in with the above exception by doing something like:

```python
if not user_event or "participants" not in user_events:
    raise ...
```

Either something like that or the .get-method on the dict should work I think🙌